packages/atlas-service/main.d.ts

```diff
@@ -0,0 +1 @@
+export * from './dist/main.d';
```

Comment:
In theory similar behavior can be achieved without all those separate files, only by setting `moduleResolution` to `NodeNext` / `Node16` in our shared ts config, but this breaks type resolution for a ton of packages we have a dependency on (a lot of them have subtle issues in their `exports` definitions), so I'm doing this in the "old" cjs way